### PR TITLE
Page/bespoke importer

### PR DESCRIPTION
**Pull request by yegor92**

### CODE REVIEW

File: src/components/SettingsForm/components/FieldContainer.tsx [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
-import FieldElement from './FieldElement';
+import styled from 'styled-components';
 
+import FieldElement from './FieldElement';
 import { generateElementInfo } from './helpers/generateElementInfo';
 import { generateTitleAndDescription } from './helpers/generateTitleAndDescription';
 
-
+const SubText = styled.span`
+  position: absolute;
```
Comment (pxzdev) [inline src/components/SettingsForm/components/FieldContainer.tsx]: removed position: absolute styles in 250 PR

File: src/pages/Bespoke/Importer/styles.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    left: 0;
+    bottom: 0;
+    height: 70px;
+    display: flex;
+    align-items: center;
+  }
+`;
+
+export const Group = styled.div`
+  flexdirection: column;
```
Comment (pxzdev) [inline src/pages/Bespoke/Importer/styles.tsx]: unknow css rule

File: src/components/SettingsForm/components/FieldContainer.tsx (hunk 1 above)
Comment (yegor92) [reply src/components/SettingsForm/components/FieldContainer.tsx]: SubText is validation  text. so can't remove.

File: src/pages/Bespoke/Importer/styles.tsx (hunk 2 above)
Comment (yegor92) [reply src/pages/Bespoke/Importer/styles.tsx]: Fixed

File: src/components/SettingsForm/components/FieldContainer.tsx (hunk 1 above)
Comment (yegor92) [reply src/components/SettingsForm/components/FieldContainer.tsx]: Sorry. it is not validation text.
Screen Shot 2020-02-03 at 5 35 26 PM

Comment (yegor92) [reply src/components/SettingsForm/components/FieldContainer.tsx]: SubText is this.

Comment (pxzdev) [reply src/components/SettingsForm/components/FieldContainer.tsx]: Im talking about position: absolute styles. We should not to use position styles rules.

**Pull request merged**